api/src/config.rs

```diff
@@ -820,8 +820,8 @@ pub struct RafsConfigV2 {
     #[serde(default = "default_rafs_mode")]
     pub mode: String,
     /// Batch size to read data from storage cache layer.
-    #[serde(default = "default_batch_size")]
-    pub batch_size: usize,
+    #[serde(rename = "batch_size", default = "default_user_io_batch_size")]
```

Comment:
Any docs need to be updated? mainly update the comment of the field.

Reply:
Added to `docs/nydusd.md`
For anywhere else, it is not provided since it is an optional field.